internal/tiltfile/live_update_test.go

```diff
@@ -134,40 +134,47 @@ live_update('gcr.io/bar',
 
 func TestLiveUpdateBuild(t *testing.T) {
 	type testCase struct {
-		name, buildCmd string
-		assert         func(f *fixture, expectedLu model.LiveUpdate)
+		name, tiltfileCode string
+		assert             func(f *fixture, expectedLu model.LiveUpdate)
 	}
 
 	dbManifestOpt := func(f *fixture, expectedLu model.LiveUpdate) {
-		f.assertNextManifest("foo", db(image("gcr.io/foo"), expectedLu))
+		f.assertNextManifest("foo", db(imageNormalized("foo"), expectedLu))
+	}
+	dbDefaultRegistryManifestOpt := func(f *fixture, expectedLu model.LiveUpdate) {
+		i := imageNormalized("foo")
+		i.deploymentRef = "gcr.io/foo"
+		f.assertNextManifest("foo", db(i, expectedLu))
 	}
 	cbManifestOpt := func(f *fixture, expectedLu model.LiveUpdate) {
-		f.assertNextManifest("foo", cb(image("gcr.io/foo"), expectedLu))
+		f.assertNextManifest("foo", cb(imageNormalized("foo"), expectedLu))
 	}
 
 	tests := []testCase{
-		{"docker_build", "docker_build('gcr.io/foo', 'foo')", dbManifestOpt},
-		{"custom_build", "custom_build('gcr.io/foo', 'docker build -t $TAG foo', ['foo'])", cbManifestOpt},
+		{"docker_build", "docker_build('foo', 'foo')", dbManifestOpt},
```

Comment:
nice! probably good to have a non-normalizable image name (`gcr.io/.../foo` etc.?) w/o default registry as part of this test?

Reply:
yeah

this was the straw that broke the table test's back, so I ended up restructuring all these tests

Reply:
RIP camel

☠️ 🐫 😢